README.md

```diff
@@ -111,6 +111,7 @@ holdFee:
 ### Run
 
 * Clone this repository
+* `cd circuitbreaker/`
```

Comment:
I thought people familiar with `git clone` know to go into that dir, but it doesn't hurt to make it explicit. Thx.